tests/e2e-instrumentation/instrumentation-java-tls/00-install-collector.yaml

```diff
@@ -41,3 +41,6 @@ spec:
         traces:
           receivers: [otlp]
           exporters: [debug]
+        metrics:
```

Comment:
i was going to attempt the TLS test, but it seemed way too hard to do on a first-pass